stockflux-securities-master/src/components/InputForm.js

```diff
@@ -125,14 +149,20 @@ const InputForm = ({ match }) => {
           <div className="input-submit-button-container">
             <div
               className={
-                (inProgress ? "in-progress " : "") + "input-submit-button"
+                (formState === stateEnum.sending ? "in-progress " : "") +
+                "input-submit-button"
               }
             >
               <button>{match.params.securityId ? "Edit" : "Create"}</button>
             </div>
           </div>
         </form>
       )}
+      {!!messages &&
+        formState !== stateEnum.sending &&
```

Comment:
An alert should *only* be shown if the state is error, or success right?

    {messages.length > 0 && [stateEnum.success, stateEnum.error].indexOf(formState) > -1 && (
      <Alert type={formState} messages={messages} />
    )}

Reply:
good point, removed this line now